### PR TITLE
Upgrade to rails 6

### PR DESCRIPTION
**Pull request by sandbergja**
Upgrades blacklight to 7.3 (the lowest version that is compatible with rails 6).

Also adds some system tests, which required changes to the CI config and CI solr config.

Blacklight removed the saved search functionality in 7.0 (https://github.com/projectblacklight/blacklight/pull/1736), and we decided not to reimplement it for this app, since it has never been used.

### CODE REVIEW

**Comment by sandbergja** [comment]
Rebased to include the seed data from #249

**Comment by kevinreiss** [comment]
I checked this out in staging and functionality generally looked good. Only two comments:

1. There seems to be some spacing issues with facet names. The wrapping is off. 
2. There also aren't any facet "breadcrumbs" appearing when you select a facet. 

2 may be expected behavior in this version of Blacklight, I don't know. 

Screen Shot 2022-04-21 at 2 47 52 PM
Screen Shot 2022-04-21 at 2 47 10 PM

**Comment by sandbergja** [comment]
I reduced the spacing and got the constraints/breadcrumbs back.

The bookmarks feature was also missing, so I got that back too.

I noticed that the show page isn't displaying the sidebar with the email, sms, cite, and bookmark actions, though.  Still need to investigate that.

**Comment by carolyncole** [comment]
@sandbergja was the link color change expected
New:
Screen Shot 2022-05-09 at 3 13 25 PM
Old:
Screen Shot 2022-05-09 at 3 13 31 PM

**Review by carolyncole — CHANGES_REQUESTED** [review]
The new color on the buttons is adding to the accessibility contrast issues.  We should likely go back to the old color scheme.
Screen Shot 2022-05-09 at 3 19 06 PM

**Comment by carolyncole** [comment]
One other change I noted:  There are some links on the show page that are no longer clickable
Old:
Screen Shot 2022-05-09 at 3 23 59 PM
New:
Screen Shot 2022-05-09 at 3 24 09 PM

**Comment by carolyncole** [comment]
Sorry!  Another thing on the show page.  The very top header goes away:
Old:
Screen Shot 2022-05-09 at 3 26 50 PM

New:
Screen Shot 2022-05-09 at 3 27 04 PM.

File: .circleci/config.yml [hunk 1]
```diff
@@ -45,13 +47,8 @@ jobs:
...
           name: Install dependencies
           command: bundle check --path=vendor/bundle || bundle install --path=vendor/bundle --jobs 4 --retry 3
-      - run:
-          name: Install Chrome
-          command: |
-            wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
-            sudo sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
-            sudo apt-get update
-            sudo apt-get -y install google-chrome-stable
+      - browser-tools/install-chrome
```
Comment (carolyncole) [inline .circleci/config.yml:50]: 🎉

**Comment by sandbergja** [comment]
Thanks for catching all this, @carolyncole!  I've resolved everything except for the new green color for applied facets.  It's something that was added in the new Blacklight version.  Maybe we can discuss it a bit at tomorrow's meeting?

**Review by carolyncole — APPROVED** [review]
I'm not merging, since we want to have a discussion on the green.  I am approving though because the accessibility issue is gone! 🎉 
Screen Shot 2022-05-09 at 6 11 59 PM

**Review by kevinreiss — APPROVED** [review]

**Pull request merged**